backend/mora/service/orgunit.py

```diff
@@ -477,7 +497,18 @@ def get_orgunit(unitid):
     '''
     c = common.get_connector()
 
-    r = get_one_orgunit(c, unitid, details=UnitDetails.FULL)
+    details = flask.request.args.get("unitdetails", UnitDetails.FULL)
```

Comment:
Needs documentation if retained, and should be called the same as the relevant argument to `/e/`.